src/solvers/refinement/refine_arrays.cpp

```diff
@@ -130,7 +130,7 @@ void bv_refinementt::freeze_lazy_constraints()
     for(std::set<symbol_exprt>::const_iterator it=symbols.begin();
```

Comment:
While we are changing this loop, can we make it a ranged-for? (and the previous loop as well)